app/src/main/java/org/thoughtcrime/securesms/ui/Components.kt

```diff
@@ -130,13 +135,12 @@ fun ItemButtonWithDrawable(
         text = stringResource(textId),
         icon = {
             Image(
-                painter = rememberDrawablePainter(drawable = context.getDrawable(icon)),
+                painter = rememberDrawablePainter(drawable = AppCompatResources.getDrawable(context, icon)),
                 contentDescription = stringResource(contentDescription),
                 modifier = Modifier.align(Alignment.Center)
             )
         },
         colors = colors,
-        contentDescription = stringResource(contentDescription),
```

Comment:
This doesn't seem right. The `contentDescription` should apply to the button, not the image above. In fact the image should have no content description so that the screen reader reads the button overall.